### PR TITLE
Add a basic integration test

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -129,6 +136,30 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
```
Comment (ghomanyi) [inline pom.xml]: there is already a surefire plugin in line 165.

File: pom.xml [hunk 2]
```diff
@@ -129,6 +136,30 @@
...
+                <version>2.22.0</version>
+            </plugin>
+            <plugin> <!-- integration tests -->
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.0</version>
+                <executions>
+                    <execution>
+                        <phase>integration-test</phase>
```
Comment (ghomanyi) [inline pom.xml]: there is already an integration testing plugin in line 241. I don't know if it is properly configured to pick up the IT.

File: src/test/java/com/digitalasset/refapps/supplychain/SupplychainIT.java [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+
+    // Send a quote request
+    OrderedProduct orderedProduct = new OrderedProduct("Product1", 100L, date1, date2);
+    sandbox
+        .getLedgerAdapter()
+        .exerciseChoice(
+            BUYER_PARTY,
+            buyerSellerRelationshipCid.exerciseBuyerSellerRelationship_SendQuoteRequest(
+                Collections.singletonList(orderedProduct)));
+    // ??
```
Comment (ghomanyi) [inline src/test/java/com/digitalasset/refapps/supplychain/SupplychainIT.java]: ??

File: pom.xml (hunk 2 above)
Comment (gaborh-da) [reply pom.xml]: I delete that one.

File: pom.xml (hunk 1 above)
Comment (gaborh-da) [reply pom.xml]: I delete it.

File: src/test/java/com/digitalasset/refapps/supplychain/SupplychainIT.java (hunk 3 above)
Comment (gaborh-da) [reply src/test/java/com/digitalasset/refapps/supplychain/SupplychainIT.java]: It begged for a good name.

Comment (gaborh-da) [reply src/test/java/com/digitalasset/refapps/supplychain/SupplychainIT.java]: I didn't gave it any.

**Pull request merged**